### PR TITLE
Design icon to show blocking is OFF (#682)

### PR DESCRIPTION
**Pull request by ekager**

### CODE REVIEW

**Comment by ekager** [comment]
I added the new image files and added a block badging visibility method, but as we discussed in person, left finding the view with the rest!

**Comment by ekager** [comment]
@pocmo There is an svg for this icon, but it was appearing distorted on certain API levels for unknown reasons... So I switched to the pngs. I can go back and investigate the svg issue more if it will keep the app small!

**Comment by pocmo** [comment]
Okay. In this case let's land the code as-is (to get it out and tested) and let's file a new issue for updating the icon with a SVG?

In the past we had luck with other web-based SVG to Drawable XML converters that were smarter than Android Studio. There's also a list of unsupported SVG features - maybe this can help our designers to create a compatible version of the icon:
https://developer.android.com/studio/write/vector-asset-studio.html#PSD

File: app/src/main/res/layout/toolbar.xml [hunk 1]
```diff
@@ -67,16 +67,35 @@
...
-        android:layout_marginTop="8dp"
-        android:background="?android:attr/selectableItemBackgroundBorderless"
-        android:padding="4dp"
-        android:contentDescription="@string/content_description_menu"
-        android:src="@drawable/ic_menu" />
+    <FrameLayout
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:paddingTop="4dp"
+        android:paddingEnd="4dp">
```
Comment (pocmo) [inline app/src/main/res/layout/toolbar.xml]: This padding moved the menu button slightly to the left. It's almost not noticeable. :)

Maybe you can move the badge view with "margin" into position instead?

File: app/src/main/res/layout/toolbar.xml [hunk 2]
```diff
@@ -67,16 +67,35 @@
...
+                android:layout_height="18dp"
+                android:layout_gravity="top|right|center"
+                android:src="@drawable/ic_tracking_protection_16_disabled"
+                android:tint="#FFFFFF"
+                android:background="@drawable/gold_badge_background"
+                android:visibility="gone" />
+
+    </FrameLayout>
+
+
```
Comment (pocmo) [inline app/src/main/res/layout/toolbar.xml]: nit: two whitespaces

**Comment by ekager** [comment]
@pocmo believe I made the requested changes! I tried to use two different online svg to drawable xml convertors, but they were still distorted when I tested on API 22 devices.

**Comment by ekager** [comment]
SVG drawable is no longer distorted! I changed the color of the circle background as requested in the thread as well.

**Comment by pocmo** [comment]
Adding [blocked] label just because it's not in the v1.1 list. Let's try to land this after the release.

File: app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java [hunk 3]
```diff
@@ -846,4 +852,9 @@ public boolean isBlockingEnabled() {
         final IWebView webView = getWebView();
         return webView == null || webView.isBlockingEnabled();
     }
+
+    // In the future, if more badging icons are needed, this should be abstracted
+    public void updateBlockingBadging(boolean enabled){
```
Comment (sputnikci) [inline app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java]: \[Checkstyle\] ERROR: '\{' is not preceded with whitespace\.

**Comment by ekager** [comment]
So would this be ready to land in the next release @pocmo ?

**Comment by pocmo** [comment]
Let's merge it in! :)

**Pull request merged**